plugins/FlashforgeFinderIntegration/package.json

```
@@ -10,8 +10,8 @@
     "package_id": "FlashforgeFinderIntegration",
     "package_type": "plugin",
     "package_version": "1.0.0",
-    "sdk_version": "7.1.0",
-    "supported_sdk_versions": ["5.0.0", "6.0.0", "7.0.0"],
+    "sdk_version": "7.2.0",
+    "supported_sdk_versions": ["7.0.0"],
```

You'll want to write the version 7.2.0 here, not 7.0.0.

The `sdk_version` field is no longer being read as of Cura 3.6 (i.e. a long time ago). The `supported_sdk_versions` is really the only one that matters nowadays unless you still support SDK 5.0.0 or below.
```suggestion
    "supported_sdk_versions": ["7.2.0"],
```